chains/evm/contracts/test/pools/TokenPool/TokenPool.validateLockOrBurn.t.sol

```diff
@@ -29,7 +29,8 @@ contract TokenPoolV2_validateLockOrBurn is AdvancedPoolHooksSetup {
       inboundRateLimiterConfig: inboundFastConfig
     });
     vm.startPrank(OWNER);
-    s_tokenPool.setDynamicConfig(address(s_sourceRouter), minBlockConfirmation, address(0));
+    s_tokenPool.setDynamicConfig(address(s_sourceRouter), address(0));
```

Comment:
You simply added `setMinBlockConfirmation` to all places that had `setDynamicConfig` calls. Please check if both are actually required (I'd guess almost all only need one of the two)